File: src/main/java/com/kn/knwremodel/Haksa/HaksaController.java
package com.kn.knwremodel.Haksa;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.PostMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import lombok.RequiredArgsConstructor;

@RestController
@RequestMapping("/Haksa")
@RequiredArgsConstructor
public class HaksaController {
    private final HaksaService haksaS;

    @PostMapping("/request")
    public ResponseEntity request(){
        return ResponseEntity.ok(haksaS.findAll());
    }
}

File: src/main/kotlin/injea/knwremodel/Haksa/HaksaController.kt
package injea.knwremodel.Haksa

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.PostMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/Haksa")
class HaksaController(private val haksaS: HaksaService) {
    @PostMapping("/request")
    fun request(): ResponseEntity<*> {
        return ResponseEntity.ok(haksaS.findAll())
    }
}